lib/rbac/src/jwt.rs

```diff
@@ -18,6 +18,25 @@ pub struct Claims {
     /// Payload constraints.
     /// An object where each key is a JSON path, and each value is JSON value.
     pub payload: Option<PayloadClaim>,
+
+    /// Validate this token by looking for a value inside a collection.
+    pub value_exists: Option<ValueExists>,
 }
 
 pub type PayloadClaim = HashMap<JsonPath, ValueVariants>;
+
+#[derive(Serialize, Deserialize, PartialEq, Clone, Debug)]
+pub struct ValueExists {
+    pub collection: String,
+    pub key: JsonPath,
```

Comment:
We might want to check for multiple keys and values. The example would be is that we want to check for the user exists AND they have required access level

Reply:
@generall Ok for me, but we can also do it later. For now, this can be done via concatenation from the user side if refers to same collection. When we do it, we should allow different collections too.